.github/workflows/pull_request.yml

```diff
@@ -5,9 +5,28 @@ on: pull_request
 jobs:
   build:
     name: Test pull requests
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+      fail-fast: false
+    runs-on: ${{ matrix.os }}
     steps:
-    - uses: actions/checkout@master
+    - name: Set git global config (Windows)
+      if: ${{ matrix.os == 'windows-latest' }}
+      run: |
+        git config --global core.autocrlf false
+        git config --global core.eol lf
+        git config --global core.filemode false
+        git config --global branch.autosetuprebase always
+    - name: Install dependencies (Windows)
+      if: ${{ matrix.os == 'windows-latest' }}
+      run: |
```

Comment:
since this is one line we can just do `run: pip install pywin32`

same below

Reply:
Changed.